src/App/appStateStore.js

```diff
@@ -69,15 +69,18 @@ async function getCurrentUserOrganisationUnits(disableCache = false) {
     }
 
     const d2 = await getInstance();
-    const organisationUnitsCollection = await d2.currentUser.getOrganisationUnits({ paging: false });
+    const organisationUnitsCollection = await d2.currentUser.getOrganisationUnits({
+        fields: ':all,access,displayName,path,children[id,displayName,path,children::isNotEmpty,access]',
```

Comment:
Are you sure you need `:all` fields?

Reply:
Yes, it is absolutely necessary for the app to function.

I needed to override the fields passed to `getOrganisationUnits`, and only way to do that was to duplicate the `fields` found here: https://github.com/dhis2/d2/blob/master/src/current-user/CurrentUser.js#L212 so I could add `access` to that list (not sure if i should have expected to get it using `:all`, but I don't.

So this is actually done *every time* any app does a call to `d2.currentUser.getOrganisationUnits()`  without overriding the `fields` property. It's just more explicit now.